index.html

```diff
@@ -11,6 +11,11 @@
     <main id="main">
       <h1 id="title">Dr. Norman Borlaug</h1>
       <figure id="img-div">
+        <!-- <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+        /> -->
         <img
           style="display: block; max-width: 100%"
```

Comment:
I suggest that you place all the styles in the style sheet. That will make your code cleaner and more organized.